jerry-core/jcontext/jcontext.h

```diff
@@ -170,7 +170,9 @@ struct jerry_context_t
  * This part is for JerryScript which uses external context.
  */
 
-#define JERRY_CONTEXT(field) (jerry_port_get_current_context ()->field)
+#define JERRY_INIT_CURRENT_CONTEXT() \
```

Comment:
Please add doxygen comment for `JERRY_INIT_CURRENT_CONTEXT` and `JERRY_CONTEXT`

Reply:
I think the name is misleading, IMHO `JERRY_DEFINE_CURRENT_CONTEXT` or `JERRY_GET_CURRENT_CONTEXT` fits better.